src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -1485,7 +1486,7 @@ crucible_alloc_with_mutability_and_size mut sz alignment bic opts lty =
      alignment' <-
        case alignment of
          Just a -> do
-           when (a < memTyAlign) $ fail $ unlines
+           when (a < memTyAlign) $ liftIO $ printOutLn opts Info $ unlines
              [ "User error: manually-specified alignment was less than needed"
```

Comment:
We should change this message to avoid confusion if this isn't user error.

Reply:
Yes, we should definitely change the message to remove the words "User error".

Also, because we are no longer enforcing a strict minimum alignment, maybe we should change "less than needed" to "less than default for type", and below we should change "Minimum alignment" to "Default alignment".